test/unit/logic/delegate.js

```diff
@@ -926,13 +924,11 @@ describe('delegate', () => {
 				expect(
 					accountsMock.setAccountAndGet.calledWith({
 						address: sender.address,
-						u_isDelegate: 1,
 						isDelegate: 0,
 						vote: 0,
 						username: null,
-						u_username: transaction.asset.delegate.username,
 					})
-				);
+				).to.be.true;
```

Comment:
same as above.
`expect(accountsMock.setAccountAndGet).to.be.calledWith()`

Reply:
Your suggestion gives the following error `TypeError: true is not a spy or a call to a spy!`